### PR TITLE
package podman-py

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

There's some bikeshedding to be had about the name. `podman` would presumably still be free, but following the very similar situation with [docker](https://pypi.org/project/docker/) vs. [docker-py](https://anaconda.org/conda-forge/docker-py), I chose `podman-py`. Another option might be `python-podman`, like the repo itself.

It's worth noting that the newest version on PyPI (0.12.0) is older than what I'm building here, see containers/python-podman#43.

### CODE REVIEW

**Comment by h-vetinari** [comment]
Pinging the `python-podman` maintainers (based on recent [commit history](https://github.com/containers/python-podman/commits/master)) @4383 @baude @jwhonce @Phadated @rhatdan, if someone wants to share maintainership of the conda-recipe.

**Comment by h-vetinari** [comment]
> There's some bikeshedding to be had about the name. `podman` would presumably still be free, but like for docker vs. docker-py I chose `podman-py`. Another option might be `python-podman`, like the repo itself.

@conda-forge/help-python 
Does someone have an opinion about that?

**Comment by 4383** [comment]
@h-vetinari why not simply use podman?

**Comment by h-vetinari** [comment]
> @4383: @h-vetinari why not simply use podman?

Someone else from conda-forge would have to comment about the situation with docker, but presumably this was done because conda might (at some point) install `docker` itself (rather than just its python wrapper). The situation for `podman` seems very similar to me.

The namespaces for PyPI and conda are not exactly the same - another example is `graphviz`, which only installs the wrapper via pip (leaving the executable to be installed by hand by the user), whereas `conda install graphviz` installs the executable and `conda install python-graphviz` installs the wrapper (same as `pip install graphviz`).

**Comment by scopatz** [comment]
we typically use prefixes like python-podman or py-podman. docker-py is because that is what the project calls itself

**Comment by h-vetinari** [comment]
> @scopatz: we typically use prefixes like python-podman or py-podman. docker-py is because that is what the project calls itself

Thanks for chiming in - for me, this sounds like an argument in favour of `python-podman`.

**Comment by 4383** [comment]
python-podman or podman-py WFM

**Comment by h-vetinari** [comment]
I'll wait a bit more about the name if someone else chimes in. Since conda alphabetizes the installs, I can also see the appeal of `podman-py`, because `podman` then shows up where expected.

@4383, may I add you to the recipe-maintainers (like for #8681)?

**Comment by rhatdan** [comment]
I prefer podman-py.  But am fine either way.

**Comment by 4383** [comment]
@h-vetinari yes

**Comment by h-vetinari** [comment]
@4383 Thanks!

@jwhonce
Any input on the naming question? @rhatdan and myself have a slight preference for `podman-py` vs. `python-podman`, while @4383 is fine with either.

**Comment by jwhonce** [comment]
@h-vetinari I lean slightly to python-podman because that is what is on pypi. But can live with them being different.

**Comment by scopatz** [comment]
if there is a name on pypi we should definitely use that

**Comment by h-vetinari** [comment]
@jwhonce 
Not sure what you mean, but PyPI only knows [https://pypi.org/project/podman/](podman), and not [python-podman](https://pypi.org/search/?q=python-podman). 

`pip install python-podman` fails as well, while `pip install podman` works.

Not that I'm too opposed against the name `python-podman`, but I don't see how the reason you gave holds up.

**Comment by h-vetinari** [comment]
> @scopatz: if there is a name on pypi we should definitely use that

I mean, we *can* also name it `podman` directly. I thought I was following the established custom for `docker`, which has a PyPI package `docker`, but conda-package `docker-py` (not least since podman advertises as being `alias docker=podman`).

**Comment by jwhonce** [comment]
@h-vetinari Sorry, I was looking at the wrong servers.  You are correct, we went with the shortened podman for the python module when we went public.  The git repo is containers/python-podman.

**Comment by h-vetinari** [comment]
@scopatz 
Can you opine again on [this](https://github.com/conda-forge/staged-recipes/pull/8821#issuecomment-512507903) please? `python-podman` is out of the race IMO, and it would thus be `podman-py`, unless `podman` itself can reenter out of left field.

File: recipes/podman-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py==27 or win]
+
+requirements:
+  host:
+    - pbr
+    - pip
+    - python >= 3
```
Comment (scopatz) [inline recipes/podman-py/meta.yaml]: There should not be a space between the equals and the 3. `python >=3`

File: recipes/podman-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/p/podman/podman-{{ version }}.tar.gz"
+  sha256: "3d0a787fe6f5e4e5317942de066eb17f14270c9694b5409f3708712109bc09a3"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py==27 or win]
```
Comment (scopatz) [inline recipes/podman-py/meta.yaml]: It is more robust to have the selector as `[py<3 or win]`

File: recipes/podman-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+requirements:
+  host:
+    - pbr
+    - pip
+    - python >= 3
+    - setuptools
+  run:
+    - pbr
+    - psutil
+    - python >= 3
```
Comment (scopatz) [inline recipes/podman-py/meta.yaml]: same here

**Comment by scopatz** [comment]
> Can you opine again on this please? python-podman is out of the race IMO, and it would thus be podman-py, unless podman itself can reenter out of left field.

If the git repo is `python-podman` (or if anything else uses that name) I would use that name.  However, it is up to you.  The only name that's really restricted is `podman` on itself.

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
> @scopatz: If the git repo is `python-podman` (or if anything else uses that name) I would use that name. However, it is up to you. The only name that's really restricted is `podman` on itself.

Thanks for the input (esp. that `podman` is considered restricted). I want to stay with `podman-py`, both for the similarity with `docker`, but even moreso for the the correct alphabetical ordering in `conda list` (for quick answers to "is podman installed?")

**Comment by h-vetinari** [comment]
Thanks everyone!

**Pull request merged**